### ISSUE TITLE
Controller not Working

### ISSUE THREAD

**Issue by PeppyGreyskull** [Windower/Issues#1026]
On a fresh install of both Windower and FFXI, I cannot get either a DS4 or XBONE controller working whenever windower is running. It works in POLviewer with/without Windower, and FFXI without Windower, but the moment I run the game with Windower it stops working.

**Comment by RubenatorX** [comment]
Do you have "Always Enable Gamepad" turned on in windower  profile settings -> Game?

**Comment by PeppyGreyskull** [comment]
> Do you have "Always Enable Gamepad" turned on in windower profile settings -> Game?

Yeah, the controller works in all forms of ffxi except in-game windower. Non windower ffxi, non windower polviewer, and windower polviewer

**Comment by RubenatorX** [comment]
I'm asking you if you have this setting on:
![image](https://user-images.githubusercontent.com/5944121/50877611-d5d0e900-138f-11e9-89df-74910c830535.png)

**Comment by PeppyGreyskull** [comment]
Yes. In both windower and ffxi settings

**Comment by Nifim** [comment]
Stale, many users have working DS4 and XBONE controller sets. That said we do get some support questions on the topic of getting the setup to work in our discord support channel. I do not believe there is any code actions to be taken at this time.
